rotkehlchen/tests/unit/decoders/accounting_settings/test_default_settings.py

```diff
@@ -0,0 +1,338 @@
+from typing import Optional
+from unittest.mock import patch
+
+import pytest
+
+from rotkehlchen.accounting.cost_basis.base import (
+    AssetAcquisitionEvent,
+    CostBasisInfo,
+    MatchedAcquisition,
+)
+from rotkehlchen.accounting.ledger_actions import LedgerActionType
+from rotkehlchen.accounting.mixins.event import AccountingEventType
+from rotkehlchen.accounting.pnl import PNL
+from rotkehlchen.accounting.pot import AccountingPot
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.base import HistoryBaseEntry
+from rotkehlchen.accounting.structures.processed_event import ProcessedAccountingEvent
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.accounting.transactions import TransactionsAccountant
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants import ZERO
+from rotkehlchen.constants.assets import A_DAI, A_ETH
+from rotkehlchen.constants.misc import ONE
+from rotkehlchen.db.settings import ModifiableDBSettings
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.history import prices
+from rotkehlchen.types import Location, Price, Timestamp, TimestampMS, deserialize_evm_tx_hash
+from rotkehlchen.utils.misc import ts_sec_to_ms
+
+
+# The transaction hash doesn't matter in the tests in this file, so we can use any.
+EXAMPLE_TX_HASH_HEX = '0x282f632c7a2385d026068122dc415562a1c4e827fd54673671af1d40445d6b58'
+EXAMPLE_EVM_HASH = deserialize_evm_tx_hash(EXAMPLE_TX_HASH_HEX)
+EXAMPLE_ADDRESS = string_to_evm_address('0x4bBa290826C253BD854121346c370a9886d1bC26')
+
+# Some utility timestamps. They don't matter here and are used only for mocked prices.
+TIMESTAMP_1_SECS = Timestamp(1624395186)
```

Comment:
Again same thing. If no specific value is needed no need to write it in the comment. You show it by using `make_random_timestamp()`

Reply:
Will adjust the comments, but timestamps are required to be preset because they are used for getting historical prices